wd.sh

```diff
@@ -203,6 +205,28 @@ wd_add()
     fi
 }
 
+wd_addcd() {
+    local folder=$1
+    local point=$2
```

Comment:
This doesn't account for `addcd` being used with just 1 argument?

Reply:
If addcd is just 1 arg then $2 is empty string